src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -164,6 +164,12 @@ void PerformInteractionRequest::Run() {
     return;
   }
 
+  if (!IsAllChoiceSetIdsAllowed(
+          app, msg_params[strings::interaction_choice_set_id_list])) {
+    SendResponse(false, mobile_apis::Result::REJECTED);
```

Comment:
@IGapchuk propose to add log4cxx warning with message describing why this request was rejected

Reply:
@AKalinich-Luxoft Fixed in 466447c82ad8882ea67b47f96fcc10788db6351f.